### PR TITLE
Dockerize openQA tests triggering for OBS

### PR DESCRIPTION
**Pull request by eduardoj**
We want to run the triggers of openQA in a docker environment to isolate this task from the relying operating system.

Co-authored-by: David Kang <dkang@suse.com>

### CODE REVIEW

File: openqa-trigger/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM registry.opensuse.org/home/vpereirabr/dockerimages/containers/obs-tools/base:latest
+
+RUN zypper --non-interactive --quiet ref \
+ && zypper -n in curl openQA-client
+
+COPY . /home/frontend/openqa-trigger
```
Comment (hennevogel) [inline openqa-trigger/Dockerfile]: Why COPY, why not mount the checkout?

Comment (hennevogel) [reply openqa-trigger/Dockerfile]: Hm you even mount in the Makefile below....

File: openqa-trigger/Makefile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+build:
```
Comment (hennevogel) [inline openqa-trigger/Makefile]: This could easily be a `docker-compose.yml` where we can add the rest of the services in this repo right?

Comment (eduardoj) [reply openqa-trigger/Makefile]: Done. Better with the `docker-compose.yml` file, yes.

File: openqa-trigger/Dockerfile (hunk 1 above)
Comment (eduardoj) [reply openqa-trigger/Dockerfile]: Done with the `docker-compose.yml` file.

**Pull request merged**